src/main/java/ticTacToe/grid/Grid.java

```diff
@@ -10,16 +13,17 @@
 
 public class Grid {
 
-    private final ArrayList<String> cells;
+    private ArrayList<String> cells;
```

Comment:
the type on the left should be the interface type if possible - so use List rather than ArrayList.  (LSP - the L in solid)

Reply:
It would be great if we could go through this together as it seems like I'm just breaking everything when making this change... 😭